rust/kcl-lib/src/std/extrude.rs

```diff
@@ -238,25 +238,69 @@ pub(crate) async fn do_post_extrude<'a>(
         // So, there's no need to await them.
         // Instead, the Typescript codebases (which handles WebSocket sends when compiled via Wasm)
         // uses this to build the artifact graph, which the UI needs.
-        args.batch_modeling_cmd(
-            exec_state.next_uuid(),
-            ModelingCmd::from(mcmd::Solid3dGetOppositeEdge {
-                edge_id: curve_id,
-                object_id: sketch.id,
-                face_id,
-            }),
-        )
-        .await?;
+        let opposite_edge_id = args
```

Comment:
These two are no longer batched because I need the results, but I assume there's way to batch them, and than batch all of the `Solid3dGetAllEdgeFaces` using the first batch results?

Reply:
Yes. We've never done that before, but we could add it. Pretty straight-forward. But it seems like #6482 will be even better, so let's not bother.